app/static/src/componentsStatic/ParameterSlider.vue

```diff
@@ -1,40 +1,57 @@
 <template>
-  <label :id="`${name}-input`" class="col-form-label fw-bold pb-0 label">{{ name }}</label>
-  <div :id="`${name}-description`" class="description">
-    {{ description }}
+  <div class="slider">
+    <div>
+      <label class="col-form-label fw-bold pb-0 label">{{ title || par }}</label>
+      <div class="desc">{{ desc }}</div>
+    </div>
+    <div>
+      <div class="float-end value mt-2">
+        {{ value }}
+      </div>
+      <input type="range"
+             class="form-range"
+             :min="min"
+             :max="max"
+             :step="step || (max - min) / 10000"
+             :value="value"
+             @input="handleChange"/>
+    </div>
   </div>
-  <div :id="`${name}-value`" class="float-end value mt-2">
-    {{ value }}
-  </div>
-  <input type="range" class="form-range" :min="min" :max="max" :step="step || (max - min) / 10000" :value="value" @input="handleChange"/>
 </template>
 
 <script setup lang="ts">
 import { AppState } from '@/store/appState/state';
 import { RunAction } from '@/store/run/actions';
 import { RunMutation } from '@/store/run/mutations';
-import { computed } from 'vue';
+import { computed, onMounted } from 'vue';
 import { useStore } from 'vuex';
 
 const store = useStore<AppState>();
 
 const props = defineProps({
-  name: { type: String, required: true },
-  description: { type: String, required: false },
+  par: { type: String, required: true },
+  title: { type: String, required: false },
+  desc: { type: String, required: false },
   min: { type: Number, required: true },
   max: { type: Number, required: true },
-  step: { type: Number }
+  step: { type: Number, required: false }
 });
 
-const value = computed(() => store.state.run.parameterValues![props.name]);
+const value = computed(() => store.state.run.parameterValues![props.par]);
 
 const handleChange = (e: Event) => {
   const newVal = (e.target as HTMLInputElement).value;
   const oldParameterValues = store.state.run.parameterValues;
-  const newParameterValues = { ...oldParameterValues, [props.name]: parseFloat(newVal) };
+  const newParameterValues = { ...oldParameterValues, [props.par]: parseFloat(newVal) };
   store.commit(`run/${RunMutation.SetParameterValues}`, newParameterValues);
   store.dispatch(`run/${RunAction.RunModel}`);
 };
+
+onMounted(async () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const mathJax = (window as any).MathJax;
+  await mathJax?.typesetPromise();
```

Comment:
A comment here would be useful. This is waiting for MathJax to be loaded? MathJax should be included in the build, but the client app needs to configure it (as shown in index.html) in order for it to work? That definitely needs to be documented!